wikipron.py

```diff
@@ -49,6 +49,20 @@
 _PHONES_REGEX = r"\[(.+?)\]"  # FIXME: it doesn't grab anything now
 
 
+# Map from a ISO 639 code to its non-ISO 639 Wiktionary language name.
+# ISO 639-3: https://iso639-3.sil.org/sites/iso639-3/files/downloads/iso-639-3.tab  # noqa: E501
+# Wiktionary: https://en.wiktionary.org/wiki/Category:Terms_with_IPA_pronunciation_by_language  # noqa: E501
+_LANGUAGE_CODES = {
```

Comment:
Maybe you want to put something like:

`TODO: Expand this as needed to cover additional languages.`

Reply:
Added this comment at 
8f23a9e